src/reanimated2/threads.ts

```diff
@@ -164,13 +164,10 @@ export function runOnJS<A extends any[], R>(
     // reference to the original remote function in the `__remoteFunction` property.
     fun = fun.__remoteFunction;
   }
-  if (!_WORKLET) {
-    return fun;
-  }
```

Comment:
Did you try call `runOnJS` on React Thread on mobile device?

Reply:
Everything seems to be working fine (except for I had to wrap the bare use of console.log in RunOnJS from before)

Reply:
Hmm, `console.log` should be available out-of-the-box without a need to wrap it manually with `runOnJS`, can you make sure that the following piece of code gets called?

https://github.com/software-mansion/react-native-reanimated/blob/7b43e32d871860b9d65bbd89922e1dda67ea25b1/src/reanimated2/initializers.ts#L176-L183

Reply:
It's available - the issue was when I actually wanted to wrap it in `runOnJS`.